acceptable/generate_doubles.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Canonical Ltd.  This software is licensed under the
+# GNU Lesser General Public License version 3 (see the file LICENSE).
+from __future__ import print_function
+import argparse
+import json
+import textwrap
+from sys import stdout
+
+
+def print_double(api_name, api_data, stream=stdout):
+    double_name = u'%s_%s' % (api_name, "1_0")
+    double_text = textwrap.dedent(u"""\
+    {double_name} = service_mock(
+        service={service!r},
+        methods={methods!r},
+        url={url!r},
+        input_schema={request_schema!r},
+        output_schema={response_schema!r},
+    )
+    """).format(
+        double_name=double_name,
+        **api_data
+    )
+    print(double_text, file=stream)
+
+
+def print_header(stream=stdout):
+    print(textwrap.dedent(u"""\
```

Comment:
I'd prefer stream.write rather than print here, I think? Same above.

Although this is personal preference, so not pushing for it.

Reply:
Yep stream.write is simpler.